docs/index.md

```diff
@@ -1478,3 +1499,47 @@ f\zip($miles, $john); // [
                       // ]
 ```
 
+## Typeclasses
+
+`Garp\Functional` implements various typeclasses, following [The Fantasyland Specification](https://github.com/fantasyland/fantasy-land).  
+Implementing these takes time, which means this list won't be complete for a while. Pull Requests are very welcome.
+
+A typeclass is an algebraic datatype that adheres to some laws.  
+The specific implementation is up to you, as long as the laws are obeyed, the object can be used with functions in this library, and objects can be composed to form new functionality.
+
+Read the [Fantasyland Specification](https://github.com/fantasyland/fantasy-land) for in-depth details, and [Tom Harding's Fantas, Eel, And Specification](http://www.tomharding.me/fantasy-land/) for concrete implementations (in Javascript) of these types and why they're so incredibly useful.
```

Comment:
This link is already added in the first paragraph. The link to Tom Harding is enough in this paragraph in my opinion. You say "Read the..." so I would expect a peach of text not a Git repo.

Reply:
Fair enough. 🙂 
I will tweak the text!